test/production/standalone-mode/required-server-files/required-server-files.test.ts

```diff
@@ -218,13 +218,13 @@ describe('required server files', () => {
       case: 'redirect no revalidate',
       path: '/optional-ssg/redirect-1',
       dest: '/somewhere',
-      cacheControl: 's-maxage=31536000, stale-while-revalidate',
+      cacheControl: 's-maxage=31536000',
```

Comment:
Same as https://github.com/vercel/next.js/pull/76207/files#r1975906719

Reply:
This is also a "no revalidate" case, see comment above.